### ISSUE TITLE
StorageMetaData class doesn't have 'getDownloadUrl()' method

### ISSUE THREAD

**Issue by ShahnazK** [firebase/friendlychat-android#56]
as per 'friendlychat' code lab there should be a method `getDownloadUrl()` within 'StorageMetaData` class, but i couldn't find such method.
Please see here [https://codelabs.developers.google.com/codelabs/firebase-android/#7](url)

**Comment by rosariopfernandes** [comment]
You're right. According to the [release logs](https://firebase.google.com/support/release-notes/android#version_1502), that method has been deprecated in version 15.0.2 (May 2, 2018).

They just haven't updated the codelab yet. But meanwhile you can use the code provided in [the docs](https://firebase.google.com/docs/storage/android/upload-files#get_a_download_url).
